src/components/cms/BotBuilder/BotBuilder.js

```diff
@@ -21,30 +20,125 @@ import {
   readDraft,
   deleteDraft,
 } from '../../../apis/index.js';
+import styled, { css } from 'styled-components';
 
 const cookies = new Cookies();
 let BASE_URL = urls.API_URL;
 
-const styles = {
-  home: {
-    margin: '0px 10px',
-  },
-  paperStyle: {
-    width: '100%',
-    marginTop: '20px',
-    overflow: 'overlay',
-  },
-  newBotBtn: {
-    color: 'white',
-    fontFamily: 'Helvetica',
-    fontSize: '16px',
-    paddingTop: '20px',
-  },
-  heading: {
-    color: 'rgba(0,0,0,.65)',
-    padding: '20px 0px 0px 20px',
-  },
-};
+const commonHeadingStyle = css`
+  color: rgba(0, 0, 0, 0.65);
+  padding: 20px 0px 0px 20px;
+`;
+
+const Home = styled.div`
+  margin: 0px 10px;
+
+  @media (min-width: 769px) {
+    padding: 40px 30px 30px;
+  }
+`;
+
+const Paper = styled(_Paper)`
+  width: 100%;
+  margin-top: 20px;
+  overflow: overlay;
+  padding: 15px 0px;
+
+  @media (min-width: 769px) {
+    padding: 15px;
+  }
+`;
+
+const CardContent = styled(_CardContent)`
+  color: #ffffff;
+  font-family: Helvetica;
+  font-size: 16px;
+  padding-top: 20px;
+`;
+
+const H1 = styled.h1`
+  ${commonHeadingStyle}
+`;
+
+const H2 = styled.h2`
+  ${commonHeadingStyle}
```

Comment:
If both H1 and H2 are same, then why have you created 2 variables?

Reply:
One is a `h1` tag and other one is `h2` tag